staging/src/k8s.io/kubectl/pkg/cmd/config/set_credentials.go

```diff
@@ -293,6 +302,8 @@ func (o *setCredentialsOptions) modifyAuthInfo(existingAuthInfo clientcmdapi.Aut
 			// explicitly reset exec arguments
 			modifiedAuthInfo.Exec.Args = nil
 		}
+
+		modifiedAuthInfo.Exec.ProvideClusterInfo = o.execProvideClusterInfo
```

Comment:
why reset this inside the `if o.execCommand.Provided()` block? why *not* reset InteractiveMode as well?